packages/permission_handler/tizen/src/app_settings_manager.h

```diff
@@ -4,17 +4,12 @@
 #include <functional>
 #include <memory>
 
-using OnAppSettingsOpened = std::function<void(bool result)>;
-using OnAppSettingsError =
-    std::function<void(const std::string &code, const std::string &message)>;
-
 class AppSettingsManager {
```

Comment:
I guess ``AppSettingsManager`` seems to no longer has a role. How about making it a function or changing it to have a package name as a member?

Reply:
I've update PR, please review [this](https://github.com/flutter-tizen/plugins/pull/142/commits/95ec1db1a1576e16eab84659c2b9f6b74a73776f)

Reply:
It looks a lot cleaner!! :+1: